pkg/cmd/run.go

```diff
@@ -972,6 +982,22 @@ func (o *runCmdOptions) uploadAsMavenArtifact(dependency maven.Dependency, path
 	return uploadChecksumFiles(path, options, platform, artifactHTTPPath, dependency)
 }
 
+func getRelativeToWorkingDirectory(path string) (string, error) {
```

Comment:
Let's deprecate this, so we make sure to remove as soon as the root cause is fixed.

Reply:
Thanks for reviewing @squakez ! Updated, thanks !